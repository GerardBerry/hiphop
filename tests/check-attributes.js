"use hopscript"

var hh = require("hiphop");

try {
   <hh.localsignal signal="foo">
   </hh.localsignal>
} catch (e) {
   console.log(e.message);
}

try {
   <hh.emit signal="foo" sig="werr"/>
} catch (e) {
   console.log(e.message);
}

try {
   <hh.emit func=${function(){}} />
} catch (e) {
   console.log(e.message);
}
