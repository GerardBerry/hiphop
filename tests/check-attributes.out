*** SYNTAX ERROR
*** [LOC DISABLED --- run hop -g to get location]
*** Missing children.
*** SYNTAX ERROR
*** [LOC DISABLED --- run hop -g to get location]
*** Unexpected attribute: `sig`
*** SYNTAX ERROR
*** [LOC DISABLED --- run hop -g to get location]
*** Expected attribute: `signal`
